components/Button.tsx

```diff
@@ -3,21 +3,32 @@ import { FC, PropsWithChildren } from 'react';
 interface Props {
   form?: string;
   disabled: boolean;
+  variant: string;
   onClick?: () => void;
 }
 
 const Button: FC<PropsWithChildren<Props>> = ({
   form,
   children,
   disabled,
+  variant,
```

Comment:
Have this default to the one that is was before, to avoid regressions.
```suggestion
  variant = 'inputForm',
```